packages/teleport/src/Apps/AddApp/AddApp.tsx

```diff
@@ -17,10 +17,15 @@
 import React from 'react';
 import { Flex } from 'design';
 import Dialog, { DialogTitle } from 'design/Dialog';
-import useTeleport from 'teleport/useTeleport';
-import Manually from './Manually';
+
 import * as Icons from 'design/Icon';
+
+import useTeleport from 'teleport/useTeleport';
```

Comment:
isn't the `teleport/*` supposed to be grouped? line 25

also line 27-29 isn't grouped either